src/styles/utils/variables.scss

```diff
@@ -0,0 +1,5 @@
+$color: #00acdc;
+$text-color: #616070;
+$button-color: #00acdc;
```

Comment:
The `$color` and `$button-color` variables have the same value. Consider using a single variable to maintain consistency and avoid redundancy. If the intention is to have separate variables for different elements (e.g., 'color' for general use and 'button-color' specifically for buttons), this is fine as is. However, if they are intended to always be the same, consolidating them into one variable would be better for maintainability.